site/content/docs/11-deploying.md

```diff
@@ -2,7 +2,21 @@
 title: Deployment
 ---
 
-Sapper apps run anywhere that supports Node 8 or higher.
+Sapper apps run anywhere that supports Node 10 or higher.
+
+### Deploying from sapper build
```

Comment:
I feel like it's gotten a bit muddled with interleaving stuff about building on one machine vs another. What if we just cover building and running and then talk about possibly transferring the files to another machine.

### Deploying the build output

You can install the production dependencies and build Sapper with:

```bash
npm ci --prod
npm run build
```

You can now run Sapper with:

```bash
node __sapper__/build
```

If you'd like to build on one machine and run on another, you can copy the `__sapper__/build`, `static`, and `node_modules` directories. For example, you could zip those directories and then transfer the zip to another machine.

```bash
zip -r build.zip static __sapper__/build node_modules
```

Reply:
Thanks for the summary, I took a page out of this but decided to keep the title as is, since
1. Deploying the build output implies that user has already built the output beforehand
2. It has been covered in building section and assuming from the recent comments, we're trying to focus on how to run the app
3. Running `npm run build` after `npm ci --prod` would fail and exit with an error because preexisting `node_modules` are removed and sapper module as `devDependencies` would not exist

I've removed the build command and reordered the sentences a bit, please let me know again if there's more improvements to be made.